test/from-draft7/to-2019-09/schema.json

```diff
@@ -21,15 +21,15 @@
         "title": "Nested matching schema",
         "from": {
             "$schema": "https://json-schema.org/draft-07/schema",
-            "$defs": {
+            "definitions": {
                 "foo": {
                     "$schema": "https://json-schema.org/draft-07/schema"
                 }
             }
         },
         "to": {
             "$schema": "https://json-schema.org/draft/2019-09/schema",
-            "$defs": {
+            "definitions": {
```

Comment:
I think the tests in this file are invalid. On 2019-09 and later, you want to transform `definitions` to `$defs`.

Reply:
I have removed this file as it wans't supposed to be included in this PR